src/SFA.DAS.AssessorService.Application/Handlers/Search/SearchResultExtensions.cs

```diff
@@ -17,17 +17,17 @@ static class SearchResultExtensions
     {
         public static List<SearchResult> PopulateStandards(this List<SearchResult> searchResults, IStandardService standardService, ILogger<SearchHandler> logger)
         {
-            var allStandards = standardService.GetAllStandards().Result;
+            var allStandards = standardService.GetAllStandardsV2().Result;
 
             foreach (var searchResult in searchResults)
             {
-                var standard = allStandards.SingleOrDefault(s => s.StandardId == searchResult.StdCode.ToString());
+                var standard = allStandards.SingleOrDefault(s => s.ReferenceNumber == searchResult.StdCode.ToString());
```

Comment:
This looks incorrect. StdCode is usually StandardId

Reply:
Good spot, thanks. 😁